packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx

```diff
@@ -221,7 +226,34 @@ export const handleFetchAppUsageStatsDataUseEffect = (fetchAppUsageStatsData: ()
   }
 }
 
-export const DetailedTab: React.FC<DetailedTabProps> = ({ installations, developer, appUsageStats, loadStats }) => {
+export const handleFetchHttpTrafficPerDayDataUseCallback = (
+  loadHttpTrafficPerDay?: (params: HttpTrafficPerDayParams) => void,
+) => {
+  return () => {
+    if (loadHttpTrafficPerDay) {
+      loadHttpTrafficPerDay({
+        applicationId: ['4fbbb1e8-bad0-43a2-98f9-bfb9bba366e7'],
```

Comment:
Why are these dates and the applicationId hardcoded?

Reply:
It's temporary before I integrate with the filter.

Reply:
Fixed it.